mobile-app/app/screens/AppNavigator/screens/Balances/components/Announcements.tsx

```diff
@@ -15,33 +18,79 @@ export function Announcements (): JSX.Element {
   const {
     language
   } = useLanguageContext()
-  if (isSuccess && announcements !== undefined && announcements?.length > 0) {
-    const announcement = findAnnouncementForVersion(nativeApplicationVersion ?? '0.0.0', announcements, language)
-    if (announcement !== undefined) {
-      return (
-        <ThemedView
-          testID='announcements_banner'
-          style={tailwind('px-4 py-3 flex-row items-center')} light={tailwind('bg-warning-50')}
-          dark={tailwind('bg-darkwarning-50')}
-        >
-          <ThemedIcon style={tailwind('mr-2')} iconType='MaterialIcons' name='campaign' size={22} />
-          <ThemedText style={tailwind('text-xs flex-auto')} testID='announcements_text'>
-            {announcement}
-          </ThemedText>
-        </ThemedView>
-      )
-    }
+
+  const announcement = findAnnouncementForVersion(nativeApplicationVersion ?? '0.0.0', language, announcements)
+
+  if (!isSuccess || announcements === undefined || announcements.length === 0 || announcement === undefined) {
+    return <></>
+  }
+
+  if (announcement.url !== undefined && announcement.url.length !== 0) {
+    return (
+      <ThemedTouchableOpacity
+        testID='announcements_banner'
+        style={tailwind('px-4 py-3 flex-row items-center')} light={tailwind('bg-warning-50')}
+        dark={tailwind('bg-darkwarning-50')}
+        onPress={async () => await openURL(announcement.url)}
+      >
+        <ThemedIcon style={tailwind('mr-2')} iconType='MaterialIcons' name='campaign' size={22} />
+        <ThemedText style={tailwind('text-xs flex-auto')} testID='announcements_text'>
+          {`${announcement.content} `}
+          <View>
+            <ThemedIcon
+              dark={tailwind('text-darkprimary-500')}
+              iconType='MaterialIcons'
+              light={tailwind('text-primary-500')}
+              name='chevron-right'
+              size={18}
+              style={tailwind('relative -left-1 top-1')}
+            />
+          </View>
+        </ThemedText>
+      </ThemedTouchableOpacity>
+    )
   }
-  return <></>
+
+  return (
+    <ThemedView
```

Comment:
Probably we can make it as same component, to avoid having duplicate per content. Just disable the touchable opacity if there's no link

Reply:
updated to just use `<ThemedTouchableOpacity>`